README.md

```diff
@@ -64,4 +64,25 @@ The following configuration options can be specified for the bot to use a Cosmos
 - LIQUIDATOR_COSMOS_FROM
 - LIQUIDATOR_COSMOS_FROM_PASSPHRASE
 - LIQUIDATOR_COSMOS_PK
-- LIQUIDATOR_COSMOS_USE_LEDGER
\ No newline at end of file
+- LIQUIDATOR_COSMOS_USE_LEDGER
+
+#### Configuration to run the bot using a grantee account
+The bot can also be configured to run using a delegated account. With this configuration the bot will use two accounts: the granter and the grantee.
+
+- Granter account: is the account that has balances to create the liquidation order and will be associated to the position created once the liquidation order is executed
+- Grantee account: it the account used to sign the transactions broadcasted by the bot. The only requirement for the account is to have enough INJ tokens to pay for the gas fee for the transactions
+
+When running using a delegated account, the bot will send the `MsgLiquidatePosition` with the `authz` module, including it in a `MsgExecute` message,
```

Comment:
When running using a delegated account, message `MsgLiquidatePosition` is not sent directly, but wrapped inside `MsgExecute` which is executed through `authz` module's authorization mechanism.

Reply:
Changed